.github/workflows/acc-tests.yaml

```diff
@@ -0,0 +1,52 @@
+# Flow based on proposal from: https://docs.google.com/document/d/1Gz0DohBnmH2WaqrQA3JCix_m6ISVyX56nYMDdT5worM/edit?usp=sharing
+
+# use like:
+
+# name: Run Acceptance Tests
+# on: push
+# jobs:
+#   run-acceptance-tests:
+#     uses: hashicorp/vault-workflows-common/.github/workflows/acc-tests.yaml
```

Comment:
```suggestion
#     uses: hashicorp/vault-workflows-common/.github/workflows/acc-tests.yaml@main
```

suggest to use `main` as the ref so we don't have to update all our repos if the common workflow changes